python/tests/transformers/named_image_test.py

```diff
@@ -184,22 +186,58 @@ def test_prediction_decoded(self):
             # TODO: actually check the value of the output to see if they are reasonable
             # e.g. -- compare to just running with keras.
 
+    def test_featurization_no_reshape(self):
+        """
+        Run sparkDL predictor on manually-resized images and compare result to the
+        keras result.
+        """
+        imageArray = self.imageArray
+        kerasPredict = self.kerasPredict
+
+        def rowWithImage(img):
+            # return [imageIO.imageArrayToStruct(img.astype('uint8'), imageType.sparkMode)]
+            row = imageIO.imageArrayToStruct(img.astype('uint8'))
+            # re-order row to avoid pyspark bug
```

Comment:
Optional: If there's a JIRA ticket for the bug note the ticket name (e.g. "SPARK-xxxx") in this comment?